roles/tools/defaults/main.yaml

```diff
@@ -24,6 +24,15 @@ tools_mysql_versions:
   - 5.7
   - 8.0
 
+tools_postgresql_image: docker.io/perconalab/percona-distribution-postgresql
+tools_postgresql_install: "{{ 'postgresql' in groups and groups['postgresql'] | length > 0 }}"
+tools_postgresql_versions:
+  - 11.18
```

Comment:
@crimethinking why the specific minor version?

Reply:
I fixed the version at minors for now instead of `11-dev`, `12-dev`. Can always change them especially when they push official nonlab images

Reply:
I see tags such as `11`, `14`, etc, which are tags for a minor release version .. just outdated. Please check why that is the case

Reply:
@cezmunsta Evgeniy said they would update with upcoming releases